docs/jwst/master_background/arguments.rst

```diff
@@ -0,0 +1,18 @@
+.. _msb_step_args:
+
+Step Arguments
+==============
+The master spectroscopic background subtraction step has two optional arguments.
+
+``--user_background``
+  The file name of a user-supplied 1-D master background spectrum. Must be in the form
+  of a standard :ref:`x1d <x1d>` product containing a single 'EXTRACT1D' extension. When set,
+  the creation of a master background spectrum by the step is skipped. The name of the
+  supplied file is recorded in the MSTRBKGD keyword in the output product(s).
+  Defaults to ``None``.
```

Comment:
> When set, the creation of a master background spectrum by the step is skipped. The name of the
   supplied file is recorded in the MSTRBKGD keyword in the output product(s).

I think something like
```
When a user background spectrum is supplied, it is used for the subtraction
instead of a master background spectrum generated by the step, and the name
of the supplied file is recorded in the MSTRBKGD keyword in the output product(s).
```
Is a bit clearer.

Reply:
👍 fixed in next commit